css/custom.css

```diff
@@ -5,3 +5,21 @@
 .obsolete {
   background-color: #f6f6f6;
 }
+
+.inactive_row {
+  background-color: silver;
+}
+.ic-display{
```

Comment:
can we please properly lint this file? formatting is inconsistent

Reply:
Used https://css-lint.com/ to do this, increased the diff a bit, but does not matter.